### ISSUE TITLE
NTR: X-ray or CT: pneumatosis intestinalis (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8968]
New term request X-ray or CT: pneumatosis intestinalis (hausma_120913202558): Gas within intestinal wall  UMLS: 1112565

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000377
- label: Intramural intestinal gas
- parents: [Abnormal intestine morphology (HP:0002242)](https://hpo.jax.org/app/browse/term/HP:0002242)
- pmid: PMID:25400871
- def: The presence of extraluminal gas within the bowel wall.
- comment: In a symptomatic postoperative patient such a finding on plain radiology or computerized tomography (CT) is often viewed as a signal of impending or actual acute mesenteric ischaemia especially when associated with portal venous gas.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8968
- synonyms: Pneumatosis intestinalis(NONE)
